tests/test_convert_lit_checkpoint.py

```diff
@@ -0,0 +1,57 @@
+import torch
+import json
+import os
+import pytest
+
+
+@pytest.mark.parametrize("model_size", ["7b", "40b"])
+def test_copy_weights_falcon(model_size):
```

Comment:
In https://github.com/Lightning-AI/lit-gpt/blob/main/tests/test_model.py we test our implementation by

1. Loading the HF model: https://github.com/Lightning-AI/lit-gpt/blob/main/tests/test_model.py#L105-L109
2. Copying over its weights to our model: https://github.com/Lightning-AI/lit-gpt/blob/main/tests/test_model.py#L110-L114
3. Running a sample through the model and comparing the result: https://github.com/Lightning-AI/lit-gpt/blob/main/tests/test_model.py#L116-L120

I suggest that you do exactly the same, but copying our weights to the HF model.

This means you won't need to download checkpoints or any sort of configuration file

Another interesting way to test that conversion works would be to do

```python
our_state_dict = our_model.state_dict()
to_hf = convert_lit_checkpoint(our_state_dict)
to_ours = convert_hf_checkpoint(to_hf)
assert to_ours == our_state_dict
```